longshot/main.tf

```diff
@@ -0,0 +1,24 @@
+variable "papertrail_prod_destination" {}
+variable "papertrail_qa_destination" {}
+
+resource "heroku_pipeline" "longshot" {
+  name = "longshot"
+}
+
+module "longshot-qa" {
+  source = "application"
+
+  name           = "longshot-qa"
+  host           = "longshot-qa.dosomething.org"
+  pipeline       = "${heroku_pipeline.longshot.id}"
+  pipeline_stage = "staging"
+
+  email_name    = "Longshot QA"
+  email_address = "devops@dosomething.org"
+
+  database_name = "whitelabel_scholarship"
+  database_type = "db.t2.micro"
+  database_size = 5
```

Comment:
nit: add a comment with units (GB).

Reply:
Renamed this variable to `database_size_gb` for clarity in #55.